src/utils/redirectUtils.ts

```diff
@@ -1,6 +1,12 @@
-import { roomId } from "../stores/setup";
+import { Cookies } from "react-cookie";
+import { getRoomId } from "./connectionOptions";
+import { UserResult } from "../types/User";
+import { Envs } from "../types/Envs";
+import { setErrorData } from "./errorData";
+import { HttpStatusCode } from "../types/StatusCodeEnums";
 
-export const redirectToLoginPage = () => {
+const redirectToLoginPage = () => {
+  const roomId = getRoomId();
   if (import.meta.env.PROD) {
     window.location.assign(`/login?redirect=/tldraw?roomName=${roomId}`);
```

Comment:
Not part of your PR. But this is als to hiden and a information that must be placed on a more popular place.